docs/release-promotion/mobile/howto-rc.md

```diff
@@ -108,53 +89,22 @@ source bin/activate
 # paste the export line from above, you should have found a
 # promote taskid
 #   export PROMOTE_TASK_ID=...
-#   export RC_TASK_ID=...
+#   export SHIP_TASK_ID=...
```

Comment:
Flagging @escapewindow specifically for this change as I was unable to push Fennec RC 61.0rc earlier today because of this. I am using Callek's script to grab those env vars and none of those were `RC_TASK_ID`. Hence updating this with the actual values.

Reply:
I'll defer to aki here